environment/container/docker/docker.go

```diff
@@ -119,3 +126,34 @@ func (d dockerRuntime) Version() string {
 	version, _ := d.host.RunOutput("docker", "version", "--format", `client: v{{.Client.Version}}{{printf "\n"}}server: v{{.Server.Version}}`)
 	return version
 }
+
+func (d dockerRuntime) copyCerts() error {
+	log := d.Logger()
+	err := func() error {
+		dockerCertsDirHost := filepath.Join(util.HomeDir(), ".docker", "certs.d")
+		dockerCertsDirGuest := "/etc/docker/certs.d"
+		if _, err := d.host.Stat(dockerCertsDirHost); err != nil {
+			// no certs found
+			return nil
+		}
+
+		// we are utilising the host cache path as it is the only guaranteed mounted path.
+
+		dockerCertsCacheDir := filepath.Join(config.CacheDir(), "docker-certs")
+		if err := d.host.RunQuiet("mkdir", "-p", dockerCertsCacheDir); err != nil {
+			return err
+		}
+
```

Comment:
I tried testing this locally, and I see the directory gets created, but none of the subdirectories/files get populated. I might be misreading the code here, but it looks like you create the directory in the cache on the host, but never copy the host subfolders into that directory?

Reply:
You're actually right, thanks for spotting.

Reply:
fixed https://github.com/abiosoft/colima/releases/tag/v0.3.3-pre-02